rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -91,7 +91,9 @@ object Runtime {
   type Remainder = Option[Var]
   type BodyRef   = Long
 
-  class ShortLeashParams[F[_]] private (private val params: Ref[F, ShortLeashParameters]) {
+  class ShortLeashParams[F[_]] private (
+      private val params: Ref[F, Runtime.ShortLeashParams.ShortLeashParameters]
```

Comment:
What mandates the fully qualified name of `Runtime.ShortLeashParams.ShortLeashParameters`? It's quite noisy, I'd prefer the previous version.

Reply:
If that's due to IDEA painting `usnafe` red, try this:

```scala
    def unsafe[F[_]: Sync](): ShortLeashParams[F] =
      new ShortLeashParams[F](Ref.unsafe(ShortLeashParams.ShortLeashParameters.empty))
```

Reply:
yes. IDEA was complaining. I'll fix it